pages/common/pamundice.md

```diff
@@ -0,0 +1,17 @@
+# pamundice
+
+> Combine a grid of Netpbm images into one.
+> See also: `pamdice`.
+> More information: <https://netpbm.sourceforge.net/doc/pamundice.html>.
+
+- Combine the images whose names match the `printf`-style filename expression. Assume a grid of size `x_value` times `y_value`:
+
+`pamundice {{filename_%1d_%1a.ppm}} -across {{x_value}} -down {{y_value}} > {{path/to/output.ppm}}`
+
+- Assume that the tiles overlap horizontally and vertically by the specified amount:
+
+`pamundice {{filename_%1d_%1a.ppm}} -across {{x_value}} -down {{y_value}} -hoverlap {{value}} -voverlap {{value}} > {{path/to/output.ppm}}`
```

Comment:
So it doesn't support a path as an argument?

Reply:
To be honest, i'm not entirely sure whether or not it does. It is not included in the Netpbm bundle on my computer, so I can't test.